programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -1539,11 +1561,13 @@ static struct option run_options[] = {
         { "refresh", required_argument, NULL, 'r'},
         { "dns-ip-range", required_argument, NULL, 'd'},
         { "dns-upstream", required_argument, NULL, 'u'},
+        { "disable-intercepts", optional_argument, NULL, 'h'},
```

Comment:
The dns-related arguments will not be used when intercepts are disabled, so it's confusing and misleading for them to be `required_argument`s.

I think the easiest way to avoid potentially conflicting arguments is to add a new top-level command like `run-host` which could accept its own command line arguments if necessary without interfering with the existing `run` arguments.

Another reason I like `run-host` is that it would blend in nicely with a `run-intercept` if we ever wanted to add an intercept-only mode, with `run` continuing to intercept and host services.

Reply:
Created a new top-level command run_host and removed the --disable-intercepts option from the run command. The run function that processes these 2 commands, is not split because it does not require major refactoring apart from 2 blocks of code. Please review and let me know if you want to change that.